docs/messaging/rabbitmq.md

```diff
@@ -17,11 +17,15 @@ described on this page.
 In RabbitMQ, the destination is defined by an *exchange* and a *routing key*.
 `messaging.destination.name` MUST be set to the name of the exchange. This will be an empty string if the default exchange is used.
 
-<!-- semconv messaging.rabbitmq(full,tag=tech-specific-rabbitmq) -->
+<!-- semconv messaging.rabbitmq(full,tag=tech-specific) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | [`messaging.rabbitmq.destination.routing_key`](../attributes-registry/messaging.md) | string | RabbitMQ message routing key. | `myKey` | Conditionally Required: If not empty. |
 | [`messaging.rabbitmq.message.delivery_tag`](../attributes-registry/messaging.md) | int | RabbitMQ message delivery tag | `123` | Conditionally Required: When available. |
+| [`network.peer.address`](../attributes-registry/network.md) | string | Peer address of the messaging intermediary node where the operation was performed. [1] | `10.1.2.80`; `/tmp/my.sock` | Recommended |
+| [`network.peer.port`](../attributes-registry/network.md) | int | Peer port number of the network connection. | `65123` | Recommended |
```

Comment:
Maybe worth overriding description to match `network.peer.address`, e.g. “Peer port of the messaging intermediary node where the operation was performed”